config/templates/csv-template.yaml

```diff
@@ -0,0 +1 @@
+../../hack/templates/csv.yaml
```

Comment:
Meh, why not just move it?

Reply:
in the scope of the migration, it is better to be able to still generate the CSV both ways (with common and with hack scripts). once good, we should move the file for good.

Reply:
I don't think I agree with that philosophy. You would need to switch between them with a PR anyway, and a revert is just as easy as anything else. And while it *may* be possible in some circumstances/projects to have both mechanisms workable at once, it won't be easy to accomplish; and it will be even harder to support/prove. Then to "complete" the migration we have to do the work anyway.

IMHO it's easy to prove that the migration has worked -- by comparing the bundle files before/after -- so it should be all or nothing.